extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/tableExtractors.scala

```diff
@@ -69,7 +76,9 @@ object TableExtractor {
       getOwner(table)
     } catch {
       // Exception may occur due to invalid reflection or table not found
-      case _: Exception => None
+      case e: Throwable =>
+        logDebug("Extract owner from table failed.", e)
```

Comment:
```suggestion
        logWarning("Extract owner from table failed.", e)
```

Reply:
debug is better, see https://github.com/apache/kyuubi/pull/6634#discussion_r1724322878